web-app/src/smart-lock-keys/api/smart-lock-key.schemas.ts

```diff
@@ -33,3 +33,11 @@ export const CreateSmartLockKeyRequestDtoSchema = z.object({
 export type CreateSmartLockKeyRequestDto = z.infer<
   typeof CreateSmartLockKeyRequestDtoSchema
 >;
+
+export const OpenSmartLockWithKeyRequestDtoSchema = z.object({
```

Comment:
```suggestion
export const ManageSmartLockWithKeyRequestDtoSchema = z.object({
```
we should be able to use this both to open and close a smart lock with key, right?

Reply:
I know, but if in the future we want to pass different parameter? Moreover with nuki we are able to track the state(lock/unlocked), but what if another provider not let us do it?